apm-agent-plugins/apm-jmx-plugin/src/main/java/co/elastic/apm/agent/jmx/JmxMetricTracker.java

```diff
@@ -30,18 +30,7 @@
 import org.stagemonitor.configuration.ConfigurationOption;
 
 import javax.annotation.Nullable;
-import javax.management.AttributeNotFoundException;
-import javax.management.InstanceNotFoundException;
-import javax.management.JMException;
-import javax.management.MBeanServer;
-import javax.management.MBeanServerDelegate;
-import javax.management.MBeanServerFactory;
-import javax.management.MBeanServerNotification;
-import javax.management.MalformedObjectNameException;
-import javax.management.Notification;
-import javax.management.NotificationListener;
-import javax.management.ObjectInstance;
-import javax.management.ObjectName;
+import javax.management.*;
```

Comment:
[minor] do not use wildcards for imports (this setting broke recently in my IDE too)

Reply:
thanks, yes config lost after a reset. Fixed in the commit